tutorials/v7/climate/climate_tutorial.C

```diff
@@ -0,0 +1,249 @@
+   cout << "\nThe count for 1993_to_2002: " << *df1993To2002.Count() << std::endl;
+   auto df2003To2013 = df.Filter(fn2003To2013, {"Year"});
+   cout << "The count for 2003_to_2013: " << *df2003To2013.Count() << std::endl;
+
+   // Create a violet histogram for 1993_to_2002
+   auto hist_1993_to_2002 = df1993To2002.Histo1D({"1993_to_2002 Average Temp", "Average Temperature: 1993_to_2002 vs. 2003_to_2013", 100, -40, 40}, "AverageTemperature");
+   hist_1993_to_2002->SetLineColor(kViolet);
+   hist_1993_to_2002->SetLineWidth(6);
+   // Create a spring-green histogram for 2003_to_2013
+   auto hist_2003_to_2013 = df2003To2013.Histo1D({"2003_to_2013 Average Temp", "Average Temperature: 1993_to_2002 vs. 2003_to_2013", 100, -40, 40}, "AverageTemperature");
+   hist_2003_to_2013->SetLineColor(kSpring);
+   hist_2003_to_2013->SetLineWidth(6);
+
+   // Create a canvas to display histograms for average temperature by season.
+   auto canvas = RCanvas::Create("Average Temperature by Season");
+   canvas->Draw<TObjectDrawable>(std::shared_ptr<TH1D>(fallHist.GetPtr()), "L");
+   canvas->Draw<TObjectDrawable>(std::shared_ptr<TH1D>(winterHist.GetPtr()), "L");
+   canvas->Draw<TObjectDrawable>(std::shared_ptr<TH1D>(springHist.GetPtr()), "L");
+   canvas->Draw<TObjectDrawable>(std::shared_ptr<TH1D>(summerHist.GetPtr()), "L");
+   // Create a legend for the seasons canvas
+   auto legend = std::make_shared<TLegend>(0.1,0.7,0.48,0.9);
+   legend->AddEntry(fallHist.GetPtr(),"fall","l");
+   legend->AddEntry(winterHist.GetPtr(),"winter","l");
+   legend->AddEntry(springHist.GetPtr(),"spring","l");
+   legend->AddEntry(summerHist.GetPtr(),"summer","l");
+   canvas->Draw<TObjectDrawable>(legend, "L");
+   canvas->Show();
+
+   // Create a canvas to display histograms for average temperature for 1993_to_2002 & 2003_to_2013.
+   auto canvas2 = RCanvas::Create("Average Temperature: 1993_to_2002 vs. 2003_to_2013");
+   canvas2->Draw<TObjectDrawable>(std::shared_ptr<TH1D>(hist_1993_to_2002.GetPtr()), "L");
+   canvas2->Draw<TObjectDrawable>(std::shared_ptr<TH1D>(hist_2003_to_2013.GetPtr()), "L");
+   // Create a legend for the two decades canvas
+   auto legend2 = std::make_shared<TLegend>(0.1,0.7,0.48,0.9);
+   legend2->AddEntry(hist_1993_to_2002.GetPtr(),"1993_to_2002","l");
+   legend2->AddEntry(hist_2003_to_2013.GetPtr(),"2003_to_2013","l");
+   canvas2->Draw<TObjectDrawable>(legend2, "L");
+   canvas2->Show();
+
+   // Logic to reconcile applying ROOT 6 histograms on a ROOT 7 canvas
+   // Synchronous, wait until painting is finished
+   canvas->Update(false,
+   [](bool res) { std::cout << "First Update done = " << (res ? "true" : "false") << std::endl; });
+   // Invalidate canvas and force repainting with next Update()
+   canvas->Modified();
+   // Call Update again, should return immediately if canvas was not modified
+   canvas->Update(false,
+   [](bool res) { std::cout << "Second Update done = " << (res ? "true" : "false") << std::endl; });
```

Comment:
Probably, this block is not required here and can be removed.
This feature is only important for live drawings update.
`RCanvas` is capable to make such update asynchronous to caller thread, therefore callback is important to understand when update is finished.
But 99% of people does not care at all

Reply:
When I redeployed without this section, no visualizations appeared on the browser canvas (at least on my local configurations). However, I am open to corrections/updates on this matter if I am overlooking something. @linev @jblomer

Reply:
@linev @jblomer Would you be able to reproduce it without the aforementioned components?

Reply:
@JohnYoon13 I can confirm that at least one call to `canvas->Update(...)` is needed to avoid a blank window. @linev what would you suggest as a correct call here to display the canvas?

Reply:
This must be removed - it is only side effect of ownership problem